test/run_readelf_tests.py

```diff
@@ -80,6 +80,15 @@ def run_test_on_file(filename, verbose=False, opt=None):
                 testlog.info('.......................SKIPPED')
             continue
 
+        # TODO(sevaa): I've checked the readelf sources, the DWARF docs,
```

Comment:
Can we do it slightly differently, like excising the the `eh_frame` section from this sample file?

Reply:
I could try and patch the instruction area of the offending FDE with NOPs. Is there a control sum in ELF?

Reply:
I was thinking just deleting the whole section like with `objcopy` (https://stackoverflow.com/questions/31453859/how-to-remove-a-specific-elf-section-without-stripping-other-symbols) or maybe there was another tool for this - I don't recall exactly.

This isn't a huge priority - fine to leave as is if this is hard. Just looking for a way to avoid more hacks in this test file if possible

Reply:
Patched. One instruction that threw binutils off replaced with two zero bytes (NOPs). Made a note in the test script.